judo-runtime-core-jsl-itest/src/test/resources/model/PrimitivesModel/Primitives.jsl

```diff
@@ -110,3 +110,25 @@ entity DefaultRequiredEntity {
     field required Integer sumEntitiesIntegerValue = ReferenceEntity!all()!sum(e | e.intAtt);
 
 }
+
+//Entity with not valid default values
+
+entity EntityWithNotValidIntegerDefaultValue {
+        field Integer integerAttr = 1234567890;
+}
+
+entity EntityWithNotValidPrecisionDefaultValue {
+        field Scaled scaledAttrWithWrongPrecision = 123456789.34;
+}
+
+entity EntityWithNotValidScaleDefaultValue {
+        field Scaled scaledAttrWithWrongScale = 2.3466;
+}
+
+// entity EntityWithNotValidStringDefaultValue {
+//         field SmallString stringAttr = "To large string literal. To large string literal.To large string literal.To large string literal.To large string literal.Too larg";
+// }
```

Comment:
If this is commented due to a bug, please add ticket reference.

Reply:
[bdfe200](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/191/commits/bdfe200f6a3608a459951711ccba6415c3ec1996)